test/source/transitionUsingParentViewGenerator.js

```diff
@@ -0,0 +1,40 @@
+var spyOnBackbone = require('../spec/functional/backboneSpy'),
```

Comment:
do you need to do anything with spyOnBackbone after requiring it?  I don't see it used below.  Same for most of the view files.